dbt/include/bigquery/macros/materializations/materialized_view/materialized_view.sql

```diff
@@ -0,0 +1,38 @@
+{% materialization materialized_view, adapter='bigquery' %}
```

Comment:
Do we have to rewrite the materialization? I think we should be able to use the default one in `dbt-core`.